lib/child_process.js

```diff
@@ -109,7 +97,6 @@ function normalizeExecArgs(command /*, options, callback*/) {
 exports.exec = function(command /*, options, callback*/) {
   var opts = normalizeExecArgs.apply(null, arguments);
   return exports.execFile(opts.file,
-                          opts.args,
```

Comment:
How does this work?  `execFile()` ends up calling `spawn()` but I can't figure out where `opts.args` is used as the arguments array.